sdks/python/apache_beam/runners/direct/watermark_manager.py

```diff
@@ -143,13 +143,18 @@ def _refresh_watermarks(self, applied_ptransform):
             for consumer in consumers:
               self._refresh_watermarks(consumer)
 
-  def extract_fired_timers(self):
+  def extract_all_timers(self):
```

Comment:
<!--thread_id:cc_153932401_t; commit:2ab61a1a97dda2326dc339f5f9dda40a04a47d9f; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Can we use the previous method name?  This name is misleading because it isn't we still only extract fired timers.

<!--section:body-->
I meant to do the following changes:
WatermarkManager's extract_fired_timers --> extract_all_timers (meaning for all transforms)
_TransformWatermarks' extract_fired_timers --> extract_transform_timers (meaning per transform).
I meant to accomplish two things: 1) differentiate between "for all" and "per transfer" and 2) we are no longer just extracting the fired timers. I understand 2) is not crucial, as the other return is basically a flag.
Another (verbose) option is:  extract_fired_timers and extract_fired_timers_per_transform.